### DISCUSSION TITLE
Hubs Login Idle Logout Time

### DISCUSSION THREAD

**Discussion by lukewhitingdev** [mozilla/hubs#4627] [Hubs Cloud Questions]
Hi, 
I was wondering if there was a way to set the logout idle time for the hubs server? Also if there isn't a way to change it, is there a default amount of time that it takes for hubs to logout a idle user?

Just to clarify since there is a idle detector inside the hubs room code, I mean the overall hubs login that is generated from the magic link (e.g. idle time outside of room).

Sorry if this is confusing, I can clarify anything in the comments

**Comment by pattersonbl2** [comment]
Could you give us an example of what you're trying to do ?  I am not understanding the use case.

**Reply by lukewhitingdev** [reply]
Sure np, 

Basically I have a web-front end which i use to send people to hubs rooms. Currently it require users to be logged in on the hubs side (from the spoke / hubs magic links). From testing it seems that after not interacting with the hubs server for a while the user is automatically logged out of hubs. Just wondering if there was a way to edit this auto-logout functionality?

**Comment by pattersonbl2** [comment]
I don't know if a way to toggle this feature off.  I could reach out to the community discord to see if someone has been able to find the section in the code that controls this.  I will reach out to my team to see if they have any idea if this can be done.
